packages/mobile/README.md

```diff
@@ -49,15 +49,22 @@ The app uses [React Native][react native] and a geth [light node][light node].
 
 **You must have the [wallet] monorepo successfully set up and built before setting up and running the mobile wallet.** To do this, follow the [setup instructions][setup].
 
-To do this, follow the [setup instructions][setup].
-
 Next, install [watchman][watchman].
 
 ```bash
 # On a mac
 brew install watchman
 ```
 
+#### Google Cloud
+You will need to be added the team keyring on GCP so you can decrypt secrets in the repo. (Ask for an invite to `celo-testnet`.) 
+
+Once you have access, install Google Cloud by running `brew install google-cloud-sdk`.
+Follow instructions [here](https://github.com/celo-org/bootnode/blob/4bdd7e7ecb91db54dc2a307ec45887d73aa75394/engsetup/README.md)
+for logging in with Google credentials.
+
+Then run `yarn keys:decrypt` from the wallet repo root. You should see something like this: `Encrypted files decrypted`
```

Comment:
This is useful to keep in as a way to test that things are working as expected. However, this command is automatically done as part of the `postinstall` script so worth noting that you shouldn't have to manually call the `keys:decrypt` script on an ongoing basis :)